more_autograding_examples/notebook_expected_string/config/config.json

```diff
@@ -1,7 +1,6 @@
 {
     "required_capabilities": "notebook",
 
-    "hide_submitted_files" : true,
```

Comment:
This was an important intentional feature.  Don't just remove it.  If it's broken it should be fixed.  It is very different from the ubuntu custom -- don't put them in the same PR.